packages/conf-adwaita-icon-theme/conf-adwaita-icon-theme.1/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "2.0"
+maintainer: "7895506+MSoegtropIMC@users.noreply.github.com"
+homepage: "https://github.com/adwaita/adwaita-icon-theme"
+bug-reports: "https://gitlab.adwaita.org/adwaita/adwaita-icon-theme/issues"
+authors: "GNU Project"
+license: ["LGPL-3.0-only" "CC-BY-SA-3.0"]
+# E.g. on Fedora the system package does not contain a .pc file.
+# Testing for this folder seems to be more portable than using system packager commands as fallback.
+build: [ "sh" "-c" "pkg-config --short-errors --print-errors adwaita-icon-theme || [ -d /usr/share/icons/Adwaita ]"]
+depends: [
+  "conf-pkg-config" {build}
+]
+depexts: [
+  # The list of supported os-family names can be derived from the opam sources
+  #   https://github.com/ocaml/opam/blob/6aefe95e60084e63d01b1c7c028b6b77de1f839f/src/state/opamSysInteract.ml#L84-L123
+  # The package name for many Linux/BSD/macOS/cygwin/msys platforms can be searched here
+  #   https://repology.org specifically https://repology.org/project/adwaita-icon-theme/versions
+  #   Not listed above but supported by opam are suse, oraclelinix, ol, rhel
+  [ "adwaita-icon-theme" "adwaita-icon-theme-dev" ] {os-family = "alpine"}
+  [ "adwaita-icon-theme" ] {os-family = "amzn"}
```

Comment:
I see that there is a new `amzon` and there is no longer `ol`. Just curious, I have no idea what either of these is :P

Reply:
I got this from:
(https://github.com/ocaml/opam/blob/a44bdd5e12585179167ba62602f537484c04c268/src/state/opamSysInteract.ml#L103-L117)
but I am not sure if I should take the strings which go in or if there is a back translation of `type families`.

The `suse, oraclelinix, ol, rhel` variants I did not list because (https://repology.org/project/adwaita-icon-theme/versions) does not list package names for these variants, possibly because they are commercial.